### PR TITLE
Add MLflow monitoring

### PR DESCRIPTION
**Pull request by gsgh3016**
- monitoring loss and accuracy
- add model signature and show model's id
- set FastAPI server init
- test client code with directory

### CODE REVIEW

File: model_train.py [hunk 1]
```diff
@@ -94,12 +107,29 @@ def main():
...
+            test_loss, accuracy = test(model, device, test_loader)
+            
+            mlflow.log_metric("train_loss", train_loss, step=epoch)
+            mlflow.log_metric("test_loss", test_loss, step=epoch)
+            mlflow.log_metric("test_accuracy", accuracy, step=epoch)
+        
+        input_example = X_test_tensor[0:1].to(device)
+        input_example_in_numpy = input_example.cpu().numpy()
+        output_example = model(input_example).detach().cpu().numpy()
+        signature = infer_signature(input_example.cpu().numpy(), output_example)
```
Comment (gsgh3016) [inline model_train.py:128]: infer_signature 함수에 인자를 넘겨줄 때는 numpy 혹은 dataframe

File: model_train.py [hunk 2]
```diff
@@ -94,12 +107,29 @@ def main():
...
+            train_loss = train(model, device, train_loader, optimizer, epoch)
+            test_loss, accuracy = test(model, device, test_loader)
+            
+            mlflow.log_metric("train_loss", train_loss, step=epoch)
+            mlflow.log_metric("test_loss", test_loss, step=epoch)
+            mlflow.log_metric("test_accuracy", accuracy, step=epoch)
+        
+        input_example = X_test_tensor[0:1].to(device)
+        input_example_in_numpy = input_example.cpu().numpy()
+        output_example = model(input_example).detach().cpu().numpy()
```
Comment (gsgh3016) [inline model_train.py:127]: detach 지우고 코드 돌려보기
